dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -1936,6 +1947,16 @@ $modal-no-icon-gap: 22px;
   min-width: 100px;
 }
 
+#hint-button {
+  background-color: $lightest_gray;
+  border: 1px solid $light_gray;
+  color: black;
+
+  svg {
+    margin: -12px -6px;
+  }
+}
+
```

Comment:
Could both these blocks of css live as inline styles in the Jsx component instead?